File: python-p/L1-031.py
##L1-031 到底是不是太胖了
'''
2021.11.2
21.49.08
By Xcl
'''
n = int(input())
a = []
for i in range(n):
    temp = input().split()
    c = int(temp[0])
    b = int(temp[1])
    b2 = (c - 100) * 1.8
    if abs(b2 - b) < b2 * 0.1:
        a.append('wan mei!')
    else:
        if b2 >b:
            a.append('tai shou le!')
        else:
            a.append('tai pang le!')
for i in a:
    print('You are ' + i)

File: java-i/L1-031.java
//L1-031 到底是不是太胖了
/*
2021.11.2
21.49.08
By Xcl
*/
import java.util.Scanner;
public class Main {
  public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);
    int t=sc.nextInt();
    int[][] s=new int[t][2];
    for(int i=0;i<t;i++){
      for(int j=0;j<2;j++){
        s[i][j]=sc.nextInt();
      }
    }
    for(int i=0;i<t;i++){
      double z=s[i][1];
      double weight=(s[i][0]-100)*0.9*2;
      if(Math.abs(z-weight)<weight*0.1){
        System.out.println("You are wan mei!");
      }else if((z-weight)<weight*0.1){
        System.out.println("You are tai shou le!");
      }else{
        System.out.println("You are tai pang le!");
      }
    }
  }
}
